relay-server/src/services/project.rs

```diff
@@ -796,59 +796,4 @@ mod tests {
             assert_eq!(outcome.quantity, expected_quantity);
         }
     }
-
-    #[test]
-    fn test_track_nested_spans_outcomes_span_quota() {
```

Comment:
This test relied on a previous bug where indexed categories from quotas with limit 0 were still enforced.